### DISCUSSION TITLE
Help for date and hour

### DISCUSSION THREAD

**Discussion by thibautbrico** [RomRider/apexcharts-card#139] [Q&A]
Hello
I would like to know how to get an apex card on home assistant that starts at 9am and ends at 6pm (Paris time) and not on Saturday and Sunday please.
I have searched on the internet and I did not find any information.

**Comment by RomRider** [comment]
I'm not sure I understand the request. Do you want "partial" time coverage (9-18 only) while displaying Monday to Friday 5 days in one single chart?
If that is what you are trying to achieve, I'm afraid this is not possible.

**Reply by thibautbrico** [reply]
I would like my card to start at 9am and end at 6pm and not on weekends

**Reply by RomRider** [reply]
So you want in a single chart:
* mon: 9-6pm
* tue: 9-6pm
* ...
* fri: 9-6pm

Correct?

If so, that is not possible, unless you display multiple charts, one for each day.

You could however do: Monday 0.00 to Friday 23.59

**Reply by thibautbrico** [reply]
yes please

**Reply by RomRider** [reply]
Not possible, sorry.

**Reply by thibautbrico** [reply]
Ah pity. But it will be a very good suggestion for the next wager.
Thanks a lot for creating this great card.
